adapters/repos/db/vector/hnsw/config_test.go

```diff
@@ -91,7 +91,7 @@ func Test_UserConfig(t *testing.T) {
 		input    interface{}
 		expected UserConfig
 	}
-
+	// these are the tests, I will have to add a test for manhattan distance as well
```

Comment:
This comment can be removed

Reply:
Yes, sure. Forgot to remove these, used them as reference while adding 😂.